scripts/qa_crowbarsetup.sh

```diff
@@ -3007,6 +3008,11 @@ function oncontroller_testsetup()
         if iscloudver 5plus; then
             /usr/bin/tempest-cleanup --init-saved-state || :
         fi
+        if [[ "$defcore_url" ]]; then
+           wget --no-check-certificate -Odefcore.txt "$defcore_url"
+           # this will overwrite the default temperstoptions
+           $tempestoptions="-- --load-list defcore.txt"
```

Comment:
I overwrite because the default are -t -s and it does not make to add smoke tests to defcore tests.